theme/images/sources/general/logotype-krizovatka.svg

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
```

Comment:
To samé jako vedle. SVG by mělo být optimalizovanější.

To svg nevypadá moc optimalizovaně. Inkscpae by měl mát nějakou volbu „export for web“ nebo „optimalized svg“ nebo tak něco a pak je ještě dobré to prohnat přes [svgo](https://github.com/svg/svgo).